hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -2459,6 +2459,24 @@
     <value>0</value>
   </property>
 
+  <property>
+    <description>Provide local directory which includes the related jar file as well as all the dependencies’
+      jar file. We could specify the single jar file or use ${local_dir_to_jar}/* to load all
+      jars under the dep directory.</description>
+    <name>yarn.nodemanager.aux-services.%s.classpath</name>
+    <value>NONE</value>
+  </property>
+
+  <property>
+    <description>Normally, we do not need to set this configuration. The class would be loaded from customized classpath
+      if it does not belongs to system-classes. For example, by default, the package org.apache.hadoop is in the
```

Comment:
Nit: typo, belongs -> belong. 

And following along the lines of the first comment: 

> For example, by default, the package org.apache.hadoop is in the system-classes, if your class CustomAuxService is in the package org.apache.hadoop, it would not be loaded from customized classpath. To solve this, either we could change the package for CustomAuxService, or configure  our own system-classes which exclude org.apache.hadoop.

could be:

> By default, the package org.apache.hadoop is part of the system-classes. If for example the class CustomAuxService is in the package org.apache.hadoop, it will not be loaded from the customized classpath. To solve this, either the package of CustomAuxService could be changed, or a separate system-classes could be configured which excludes the package org.apache.hadoop.